src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/ReviewHearingRequirementsConfirmation.java

```diff
@@ -35,11 +37,25 @@ public PostSubmitCallbackResponse handle(
         PostSubmitCallbackResponse postSubmitResponse =
             new PostSubmitCallbackResponse();
 
+        final AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
+
+        boolean isAcceleratedDetainedAppeal = asylumCase.read(IS_ACCELERATED_DETAINED_APPEAL, YesOrNo.class)
```

Comment:
can you check if there is an existing helper method to check if it is an ADA type in HandlerUtils. If not, please add the method there.

Reply:
Added in the HandlerUtil & updated Boolean checks to use that in the handlers